staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/named_certificates_test.go

```diff
@@ -277,7 +277,7 @@ func generateSelfSignedCertKey(host string, alternateIPs []net.IP, alternateDNS
 	template := x509.Certificate{
 		SerialNumber: big.NewInt(1),
 		Subject: pkix.Name{
-			CommonName: fmt.Sprintf("%s@%d", host, time.Now().Unix()),
+			CommonName: fmt.Sprintf("%s", host),
```

Comment:
iirc, this was added because different CA certificates with identical serial numbers and common names made some tls verification stacks choke. Was this change required to pass tests?